### PR TITLE
Dockerfile: filter gcov arcs warnings

### PR DESCRIPTION
**Pull request by junghans**
https://bugzilla.redhat.com/show_bug.cgi?id=1577508

### CODE REVIEW

**Comment by junghans** [comment]
@MrTheodor any ideas?

**Comment by MrTheodor** [comment]
@junghans no idea, seems like gcov is corrupted on fedora:

There is lots of such errors on travis:
https://travis-ci.org/espressopp/espressopp/jobs/379746722#L1657

Error writing output file '#usr#include#boost#python#object#pointer_holder.hpp.gcov'
Error writing output file '#usr#include#boost#python#object#py_function.hpp.gcov'
Error writing output file '#usr#include#boost#python#slice_nil.hpp.gcov'
Error writing output file '#usr#include#boost#python#object_core.hpp.gcov'
Error writing output file '#usr#include#boost#python#def_visitor.hpp.gcov'
Error writing output file '#usr#include#boost#python#object#forward.hpp.gcov'
Error writing output file '#usr#include#boost#python#object#make_ptr_instance.hpp.gcov'
Error writing output file '#usr#include#boost#python#class.hpp.gcov'

**Comment by junghans** [comment]
With running coverage on Ubuntu, we don't need that PR anymore.

File: .codecov.yml [hunk 1]
```diff
@@ -4,4 +4,5 @@ coverage:
   range: "70...100"
 ignore:
   - contrib/
-  - testsuite/ 
+  - testsuite/
+  - /usr/include/
```
Comment (junghans) [inline .codecov.yml]: This line is useful, so I will add that in another PR back!

**Pull request closed without merging**